### PR TITLE
Add static deployment and generation script

### PR DESCRIPTION
**Pull request by antekresic**
Users which want to use static deployment resource to deploy to a
Kubernetes cluster can use the generated script instead of needing to
depend on Helm services.

### CODE REVIEW

File: generate-deploy-script.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+fi
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DIR=$(cd $(dirname "${BASH_SOURCE}") && pwd -P)
+
+RELEASE_NAME=timescale-observability
+NAMESPACE=timescale-observability
```
Comment (blagojts) [inline generate-deploy-script.sh:14]: With timescale-observability as the release name, some of the resources may end up having names > 63 characters and k8s will not deploy them
With all the resources already in the timescale-obs namespace we may opt for a smaller release name

Comment (antekresic) [reply generate-deploy-script.sh:14]: Ah didn't know that... So we should be using `timescale-obs`? I guess `ts-obs` is to short :grinning:

**Comment by ismailyenigul** [comment]
It might be much better to provide  values.yml parameter to  ```helm template``` another customization in addition to the namespace.
Example values.yml to use external timescale database 

```
timescaledb-single:
    enabled: false
timescale-prometheus:
    enabled: true
    connection:
      user: postgres
      password: pasword
      host: remote-timescaledb.domain.com
      port: 5432
    service:
    loadBalancer:
      enabled: false
```

File: generate-deploy-script.sh (hunk 1 above)
Comment (ismailyenigul) [reply generate-deploy-script.sh:14]: This is the example case for this error. Just tried with `0.1.0-alpha.3`  release
```
$ helm install timescale-observability timescale/timescale-observability --devel

Error: CronJob.batch "timescale-observability-timescale-prometheus-drop-chunk" is 
invalid: metadata.name: Invalid value: "timescale-observability-timescale-prometheus-drop-chunk": 
must be no more than 52 characters
```

Comment (cevian) [reply generate-deploy-script.sh:14]: let's do `ts-observability`?

Comment (cevian) [reply generate-deploy-script.sh:14]: but we should also fix this issue

Comment (blagojts) [reply generate-deploy-script.sh:14]: https://github.com/helm/helm/issues/6006 it seems helm is the source of the error, not k8s, a old check for older k8s versions. But they haven't removed it

**Pull request merged**